api/src/main/java/org/openmrs/module/owa/AppRequiredModule.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2004-2014, University of Oslo
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ * Redistributions of source code must retain the above copyright notice, this
+ * list of conditions and the following disclaimer.
+ *
+ * Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation
+ * and/or other materials provided with the distribution.
```

Comment:
Did you copy this from University of Oslo?

Reply:
Yes @dkayiwa, and this is because of the meaning carried in this very java doc/comment.